resources/boost/skills/tenancy/references/installation.md

```
@@ -0,0 +1,73 @@
+- `assets/TenancyServiceProvider.stub.php`
+- `assets/migrations/*`
+
+## Required Steps
+
+1. Install the package.
+
+```bash
+composer require stancl/tenancy
+```
+
+2. Run the installer non-interactively.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)